flytekit/image_spec/image_spec.py

```diff
@@ -72,6 +73,7 @@ class ImageSpec:
     entrypoint: Optional[List[str]] = None
     commands: Optional[List[str]] = None
     tag_format: Optional[str] = None
+    docker_commands: Optional[List[str]] = None
```

Comment:
In `__post_init__` can you add `docker_commands` to the `parameters_str_list` list?

Reply:
@thomasjpfan Sure! Thanks for your quick review. We added `docker_commands` to the `parameters_str_list` now.